jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaJvm.java

```diff
@@ -254,6 +254,15 @@ public String toString() {
                 '}';
     }
 
+    private String getGroupNames(List<JpaGroup> groups) {
+        String groupNames="";
+        for(JpaGroup group: groups){
```

Comment:
No interest in doing it with streams? The end result is going to be a lot cleaner - right now you'll have a leading comma in the resulting String. And right now, the way it's currently written, the space is on the wrong side of the comma.

Check it out:
`String s = list.stream().map(Object::toString).collect(Collectors.joining(","));`
https://stackoverflow.com/questions/24882927/using-java8-to-convert-a-list-of-object-into-a-string-obtained-from-the-tostring